streams_chunks_test.go

```diff
@@ -0,0 +1,412 @@
+package lksdk
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+	"unicode/utf8"
+)
+
+func TestChunkUtf8String(t *testing.T) {
+	t.Run("Empty string", func(t *testing.T) {
```

Comment:
:nit: lower case `e` in `Empty`. Conventionally, we use all lower case for test names, comments, etc. Other tests in the rest of the file too.